eni/eni.go

```diff
@@ -0,0 +1,128 @@
+package eni
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/ec2"
+
+	"github.com/giantswarm/backoff"
+	"github.com/giantswarm/microerror"
+)
+
+const (
+	maxRetries    = 15
+	retryInterval = time.Second * 10
+)
+
+func (s *Service) getEniID(ec2Client *ec2.EC2) (string, error) {
```

Comment:
I'd uppercase ENI in all the functions

```suggestion
func (s *Service) getENIID(ec2Client *ec2.EC2) (string, error) {
```